src/main/java/net/mcreator/ui/laf/MCreatorTheme.java

```diff
@@ -72,19 +76,33 @@ public Color getMainTint() {
 		return MAIN_TINT;
 	}
 
-	@Nonnull protected abstract Color getBlackAccent();
+	private @Nonnull Color getBlackAccent() {
+		return colorScheme.getAltBackgroundColor();
+	}
 
-	@Nonnull protected abstract Color getDarkAccent();
+	private @Nonnull Color getDarkAccent() {
+		return colorScheme.getBackgroundColor();
+	}
 
-	@Nonnull protected abstract Color getLightAccent();
+	private @Nonnull Color getLightAccent() {
+		return colorScheme.getActionColor();
+	}
 
-	@Nonnull protected abstract Color getGrayColor();
+	private @Nonnull Color getGrayColor() {
+		return colorScheme.getForegroundColor();
+	}
 
-	@Nonnull protected abstract Color getBrightColor();
+	protected @Nonnull Color getBrightColor() {
+		return colorScheme.getBorderColor();
+	}
 
-	@Nonnull protected abstract String getBlocklyCSSName();
+	private @Nonnull String getBlocklyCSSName() {

```

Comment:
Not needed if name is same for all themes, which is now ok as files differentiate by theme folder name